src/main/java/sirius/db/jdbc/schema/SchemaUpdateAction.java

```diff
@@ -115,12 +116,13 @@ public String getError() {
     public void execute(Database db) {
         error = null;
         for (String statement : getSql()) {
-                try {
-                    OMA.LOG.FINE("Executing Schema Update: %s", statement);
-                    db.createQuery(statement).executeUpdate();
-                } catch (Exception e) {
-                    error = e.getMessage();
+            try {
+                if (!Sirius.isDev()) {
```

Comment:
Wieso wollen wir das auf Dev nicht loggen?

Reply:
dev is only true in local development systems. The only thing this will do, is filling up the startup logs in IntelliJ or in JUNIT runs...